.local.env

```diff
@@ -1,9 +1,14 @@
 NODE_ENV=development
 
 API_CORE_BASE=https://cube-creator.lndo.site/
+VUE_APP_API_URL=https://cube-creator.lndo.site/
```

Comment:
Why add the same value as another variable?

Reply:
It needs to start with `VUE_APP_` to be visible in vue. Maybe we can hack something in the webpack config to avoid the duplicated variable. I'll check.

Reply:
I tried some simple tricks but failed. 

Nothing super critical. We might never ever touch that again and take care if this ever bites us. I propose to keep it like that for now

Reply:
👍